transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -33,8 +33,8 @@
  * it only works on linux.
  */
 public final class EpollEventLoopGroup extends MultithreadEventLoopGroup {
-    static {
-        // Ensure JNI is initialized by the time this class is loaded by this time!
+    {
```

Comment:
I think the big difference here is that if the static initializer throws an exception and causes a ExceptionInInitializerError, following instantiations of that class will result in NoClassDefFoundError. whereas the instance initializer will allow you to instantiate the object multiple times even if it throws an exception.